ledgerctl.py

```diff
@@ -316,5 +318,39 @@ def upgrade_firmware(get_client, firmware_name, firmware_key, url, key):
     client.upgrade_firmware(firmware_name, firmware_key, url, key)
 
 
+@cli.command(
+    help="Perform quick onboarding from command line. Device must be reset and booted in recovery mode (press some buttons during the boot)."  # noqa
+)
+@click.argument("pin")
+@click.argument("words")
+@click.option(
+    "--id", help="Identity number to initialize", default=0, type=click.IntRange(0, 2)
```

Comment:
Range bounds are closed, as documented [here](https://click.palletsprojects.com/en/8.1.x/options/#range-options). I think the correct range here should be `click.IntRange(0, 1)`.

Reply:
2 is actually a supported value: it is a temporary identity. So `click.IntRange(0, 2)` is fine.